Code/Tools/ProjectManager/Source/UpdateProjectSettingsScreen.cpp

```diff
@@ -120,6 +131,38 @@ namespace O3DE::ProjectManager
         m_projectId->lineEdit()->setText(projectInfo.m_id);
 
         UpdateProjectPreviewPath();
+
+        QComboBox* combobox = m_projectEngine->comboBox();
+        combobox->clear();
+
+        // we use engine path which is unique instead of engine name which may not be
+        QString enginePath{};
+        if(auto result = PythonBindingsInterface::Get()->GetProjectEngine(projectInfo.m_path); result)
+        {
+            enginePath = result.GetValue<EngineInfo>().m_path;
+        }
+
+        int index = 0;
+        if (auto result = PythonBindingsInterface::Get()->GetAllEngineInfos(); result)
+        {
+            for (auto engineInfo : result.GetValue<QVector<EngineInfo>>())
+            {
+                if (!engineInfo.m_name.isEmpty())
+                {
+                    combobox->addItem(
+                        QString("%1 (%2)").arg(engineInfo.m_name, engineInfo.m_path),
+                        QStringList{ engineInfo.m_path, engineInfo.m_name });
+
+                    if (!enginePath.isEmpty() && enginePath == engineInfo.m_path)
```

Comment:
Shouldn't the enginePath comparison be wrapped in QDir to deal with any differences in separator(or path case on Windows)?

```suggestion
                    if (!enginePath.isEmpty() && QDir(enginePath) == QDir(engineInfo.m_path))
```

Reply:
Good catch, that would be safer. This code is currently relying on Python returning the same kind of paths from different functions, but I've already run into instances where that's not guaranteed the same.